package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "monitor-utility",
-  "version": "2.1.0",
-  "description": "Utility for verbose outputs of Monitor data using a CLI",
+  "version": "2.1.1",
```

Comment:
Would this be 2.1.2 if #30 gets approved and merged?

Reply:
Nope. 30 would be 2.1.2. Good catch!

Reply:
Correction. It would. 30 would be 2.1.3